modules/ossm-supported-configurations.adoc

```diff
@@ -58,6 +58,7 @@ endif::openshift-rosa[]
 == Supported configurations for Kiali
 
 * The Kiali console is only supported on the two most recent releases of the Chrome, Edge, Firefox, or Safari browsers.
+* OpenShift strategy is the only supported authentication configuration when Kiali is deployed with OpenShift Service Mesh (OSSM). OpenShift strategy controls access based on the individual's RBAC roles in OpenShift
```

Comment:
nit: Should the "OpenShift" in "OpenShift strategy" here be lowercase and possibly in "code" format since the actual configuration is all lowercase `openshift`? Also should there be a period on the end of the last sentence?

Reply:
Yes! Thank you! Good catch.